docs/cli.md

```diff
@@ -160,6 +160,18 @@ myext:
     option2: True
 ```
 
+Similarly, a [JSON] config file might look like this:
```

Comment:
`config` is [not a properly spelled word][1]. Please use the full word: `configuration`.

[1]: https://travis-ci.org/Python-Markdown/markdown/jobs/558492124#L310